content/24.vaccines-novel.md

```diff
@@ -431,52 +430,51 @@ Similar to BNT162b2, the mRNA-1273 vaccine was associated with mild-to-moderate
-Plasma from individuals double-dosed with Pfizer/BioNTech's BNT162b2 vaccine had up to a 16-fold reduction in neutralizing capacity against the Omicron variant [@doi:10.1101/2021.12.14.21267772] and a reduced efficacy (70%) [@doi:10.1056/NEJMc2119270].
-Estimates for the mRNA vaccines range from a 2-fold to over a 20-fold drop in neutralisation titers [@doi:10.1101/2021.12.13.21267748], hence the push for third and fourth doses of mRNA vaccines in many Western countries.
-A third mRNA vaccine dose does increase antibody titers, but these levels also wane with time [@doi:10.15585/mmwr.mm7107e2].
-Notably, immunocompromised individuals such as cancer patients seem to elicit a sufficient protective immune response against the Omicron variant when they have been boosted with a third dose of either mRNA vaccine, albeit a blunted response [@doi:10.1016/j.ccell.2021.12.014].
-While antibody titers do correlate with protection [@doi:10.1126/science.abm3425; @doi:10.1016/j.vaccine.2021.12.006; @doi:10.1016/j.vaccine.2021.05.063; @doi:10.1101/2021.05.21.21257572; @doi:10.1038/s41423-020-00588-2], they are not the only mechanisms of immune protection.
-For example, T cell and non-neutralizing antibody responses may be unaffected or less affected by the new VOC, and they warrant further investigation.
+Among the five cases in fully vaccinated and 11 cases in partially vaccined participants, the rate of infection by VOC was much higher than in the unvaccinated population (30% versus 10%), suggesting that the vaccine was less effective against the VOC than the index strain.
 
 The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy [@url:https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines].
-Previous research in the computational prediction of the efficacy of vaccines targeting the influenza A virus might complement efforts to monitor these types of viral outbreaks [@doi:10.1093/cid/ciy323].
-To adapt, booster shots are now recommended in many places, and vaccines in the near future will likely account for multiple variants and strains of SARS-CoV-2 [@doi:10.1038/s41591-021-01230-y].
+In general, mRNA vaccines remain highly effective against severe illness and death, but the effectiveness against infection generally has declined.
+A study monitoring infections in a Minnesota cohort from January to July 2021 estimated that the effectiveness of the Moderna vaccine fell to 86% and Pfizer to 76%, although protection against hospitalization remained at 91% and 85%, respectively [@doi:10.1101/2021.08.06.21261707].
+In July of that year, as the Delta variant became dominant in the U.S.A., these estimates all fell, to an effectiveness of 76% for Moderna and 42% for Pfizer and effectiveness against hospitalization of 81% and 75%, respectively [@doi:10.1101/2021.08.06.21261707].
+
+With the emergence of the Omicron VOC, vaccine effectiveness has likely further declined.
```

Comment:
New paragraph (old paragraph talked about nAbs but there have been additional studies since we wrote that)

Reply:
This is better